src/main/kotlin/com/atlassian/performance/tools/report/api/ShiftedDistributionRegressionTest.kt

```diff
@@ -52,11 +52,13 @@ class ShiftedDistributionRegressionTest(
 
     /**
      * Tests for equality of probability distribution shapes ignoring location
+     * Keep in mind that due to limitation of Kolmogorov-Smirnov algorithm, this test may fail to detect difference
+     * when sample size is low (<100), but might be too sensitive when sample size is very large
```

Comment:
Can we produce a warning in such case ?

Reply:
Let's materialize these claims via unit test cases. They can emit the warnings.
This way we'll know it's true. Also, we'll have a testing ground for future improvements, e.g. fixing these shortcomings.

Reply:
Good idea. But unfortunately, I don't have time for this right now. And we need this class in order to complete the sprint by Wednesday. I guess this is something we will have to improve later.

Reply:
Raised:
* https://ecosystem.atlassian.net/browse/JPERF-498
* https://ecosystem.atlassian.net/browse/JPERF-499